src/qt/scrolbar.cpp

```diff
@@ -99,11 +99,10 @@ void wxScrollBar::SetScrollbar(int position, int WXUNUSED(thumbSize),
     // Configure the scrollbar
     if (range != 0 )
     {
+        wxQtEnsureSignalsBlocked blocker(m_qtScrollBar);
         m_qtScrollBar->setRange( 0, range - pageSize );
         m_qtScrollBar->setPageStep( pageSize );
-        m_qtScrollBar->blockSignals(true);
         m_qtScrollBar->setValue( position );
-        m_qtScrollBar->blockSignals(false);
         m_qtScrollBar->show();
```

Comment:
I have no idea if it matters, but now signals are also blocked for `set{Range,PageStep()}` and `show()` too whereas previously they were disabled after and re-enabled before calling these functions. Is this intentional? If you're not sure neither, I'd rather put the `blocker` creation in its own block containing just `setValue()` call.